### PR TITLE
feat(route): add Uber Engineering Blog

### PR DESCRIPTION
**Pull request by hulb**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/uber/blog
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [x] 反爬/频率限制 anti-bot or rate limit?
  - [x] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [x] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note
The source site uses strict anti-scraping policy. I only use asyncPool to bypass it, but there is possibility that some articles are not full-text scrapped.

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Since this is a WordPress-based site. Have you tried its API yet?
i.e., https://blogapi.uber.com/wp-json/blog/v1/data?page=1&parent=pittsburgh&slug=engineering and https://blogapi.uber.com/wp-json/blog/v1/data?slug=introducing-workflowguard

File: lib/v2/uber/radar.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'www.uber.com': {
```
Comment (TonyRL) [inline lib/v2/uber/radar.js:2]: You need to specify the root domain only.
```suggestion
    'uber.com': {
```

File: lib/v2/uber/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/uber/blog': ['hulb'],
```
Comment (TonyRL) [inline lib/v2/uber/maintainer.js:2]: The key here **MUST** match the path in `router.js`

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: docs/en/blog.md [hunk 3]
```diff
@@ -90,6 +90,12 @@ Limit the number of entries to be retrieved by adding `?limit=x` to the end of t
 
 <RouteEn author="chazeon" example="/stratechery" path="/stratechery" />
 
+## Uber Blog
+
+### Engineering
+
+<Route author="hulb" example="/uber/blog" path="/uber/blog/:maxPage" />
```
[the remark this answers is not in the document]
Comment (hulb) [reply docs/en/blog.md:97]: > * `path` should be `/uber/blog/:maxPage?` according to `lib/v2/uber/router.js`

Actually yes, but I don't want to expose this parameter.I don't think it's useful for everyone.I make it for myself.

File: docs/blog.md [hunk 4]
```diff
@@ -332,6 +332,12 @@ username 为博主用户名，而非`xxx.hashnode.dev`中`xxx`所代表的 blog
 
 <Route author="junbaor SkiTiSu" example="/blogs/wangyin" path="/blogs/wangyin"/>
 
+## 优步
+
+### 工程技术
+
+<Route author="hulb" example="/uber/blog" path="/uber/blog/:maxPage" />
```
[the remark this answers is not in the document]
Comment (hulb) [reply docs/blog.md:339]: Actually yes, but I don't want to expose this parameter.I don't think it's useful for everyone.I make it for myself.

**Comment by hulb** [comment]
Thank you for reviewing. I need a while to modify to make it look good.

File: lib/v2/uber/blog.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
+const got = require('@/utils/got');
```
[the remark this answers is not in the document]
Comment (hulb) [reply lib/v2/uber/blog.js:1]: Thanks for the detailed guidance

File: lib/v2/uber/radar.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'uber.com': {
+        _name: 'Uber Blog',
+        www: [
+            {
+                title: 'Uber Blog - Engineering',
+                docs: 'https://docs.rsshub.app/blog.html#you-bu',
```
Comment (hulb) [inline lib/v2/uber/radar.js:7]: I'm not sure whether this doc link is correct

Comment (hulb) [reply lib/v2/uber/radar.js:7]: I have made some updates according to the comments. Let me known if mistakes exist. Thanks again.

**Comment by hulb** [comment]
I have made some updates according to the comments. Let me known if mistakes exist. Thanks again.

File: docs/blog.md [hunk 7]
```diff
@@ -332,6 +332,12 @@ username 为博主用户名，而非`xxx.hashnode.dev`中`xxx`所代表的 blog
 
 <Route author="junbaor SkiTiSu" example="/blogs/wangyin" path="/blogs/wangyin"/>
 
+## 优步
+
+### 工程技术
+
+<Route author="hulb" example="/uber/blog/:maxPage?" path="/uber/blog/:maxPage?" :paramsDesc="['获取的最多页数，默认最多获取第一页']" />
```
Comment (TonyRL) [inline docs/blog.md:339]:
```suggestion
<Route author="hulb" example="/uber/blog" path="/uber/blog/:maxPage?" :paramsDesc="['获取的最多页数，默认最多获取第一页']" />
```

File: docs/en/blog.md [hunk 8]
```diff
@@ -90,6 +90,12 @@ Limit the number of entries to be retrieved by adding `?limit=x` to the end of t
 
 <RouteEn author="chazeon" example="/stratechery" path="/stratechery" />
 
+## Uber
+
+### Engineering
+
+<RouteEn author="hulb" example="/uber/blog/:maxPage?" path="/uber/blog/:maxPage?" :paramsDesc="['max number of pages to retrieve, default to 1 page at most']" />
```
Comment (TonyRL) [inline docs/en/blog.md:97]:
```suggestion
<RouteEn author="hulb" example="/uber/blog" path="/uber/blog/:maxPage?" :paramsDesc="['max number of pages to retrieve, default to 1 page at most']" />
```

File: lib/v2/uber/radar.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'uber.com': {
+        _name: 'Uber Blog',
+        www: [
+            {
+                title: 'Uber Blog - Engineering',
```
Comment (TonyRL) [inline lib/v2/uber/radar.js:6]:
```suggestion
                title: 'Engineering',
```
No need to repeat `_name`

**Pull request merged**